src/test/dashboard/widgets/Number.test.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { shallow } from 'enzyme'
+
+import Number from '../../../components/dashboard/widgets/Number'
+import widgetData from '../../../reducers/widgets/dummy-data/dashboard-tools/number.js'
+
+describe( 'Number widget component', () => {
+  const statistic = '200'
+  const goal = '200'
+  const progress = ( statistic / goal ) * 100
+  const roundedProgress = parseFloat( progress ).toFixed( 2 )
+
+  const numberWrapper = shallow( <Number { ...widgetData } /> )
+  const calculatedProgress = numberWrapper.instance().calculateProgress()
```

Comment:
Please move the variable statement of `caluculatedProgress` into the test function that uses it, after line 42. You already did this in the test below it. This makes the test clearer and easier to read.